baseplate/clients/sqlalchemy.py

```diff
@@ -155,12 +156,46 @@ class SQLAlchemyEngineContextFactory(ContextFactory):
 
     """
 
-    def __init__(self, engine: Engine):
+    PROM_PREFIX = "bp_sqlalchemy_pool"
+    PROM_LABELS = ["pool"]
+
+    promTotalConnections = Gauge(
+        f"{PROM_PREFIX}_size",
+        "Total number of connections in this pool",
+        PROM_LABELS,
+    )
+
+    promCheckedInConnections = Gauge(
+        f"{PROM_PREFIX}_checked_in",
+        "Number of available, checked in, connections in this pool",
+        PROM_LABELS,
+    )
+
+    promCheckedOutConnections = Gauge(
+        f"{PROM_PREFIX}_checked_out",
```

Comment:
Could this be `in_use` to be consistent with thrift and memcached and `free` for `checked_in`?

Reply:
We usually say `active` for connection pools.

Reply:
Yeah, named it like this to be in line with the library vocabulary, but it does not make intuitive sense to me either. Changed it to active/idle.